<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:property name="field_of_view" value="0.02"/>

  <link name="tof_link"/>
  <joint name="tof_joint" type="fixed">
      <parent link="base_link"/>
      <child link="tof_link"/>
      <origin rpy="0 0 0" xyz="0.043 0 -0.005"/>
  </joint>

</robot>
